pkg/store/bucket.go

```diff
@@ -758,8 +763,9 @@ func blockSeries(
 			// Schedule loading chunks.
 			s.refs = make([]uint64, 0, len(chks))
 			s.chks = make([]storepb.AggrChunk, 0, len(chks))
-			for _, meta := range chks {
-				if err := chunkr.addPreload(meta.Ref); err != nil {
+			for j, meta := range chks {
+				// s is appended to res, but not at every iteration, hence len(res) is the index we need.
```

Comment:
What is `s`? (:

Reply:
`s := seriesEntry{}`, defined several lines before. Extended this comment a bit